src/parser/jobs/mnk/modules/Speedmod.js

```diff
@@ -7,6 +7,7 @@ const GREASED_LIGHTNING_STATUSES = [
 	STATUSES.GREASED_LIGHTNING_I.id,
```

Comment:
missing the rename here by the looks. probs worth doing a search+replace across the system
```suggestion
	STATUSES.GREASED_LIGHTNING.id,
```